src/Accordion.js

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import useUncontrolled from 'uncontrollable/hook';
+import { useBootstrapPrefix } from './ThemeProvider';
+import AccordionContext from './AccordionContext';
+import AccordionToggle from './AccordionToggle';
+import SelectableContext from './SelectableContext';
+import AccordionCollapse from './AccordionCollapse';
+
+const propTypes = {
+  /** Set a custom element for this component */
+  as: PropTypes.elementType,
+
+  /** @default 'accordion' */
+  bsPrefix: PropTypes.string,
+
+  /** The current active key that corresponds to the currently expanded card */
+  activeKey: PropTypes.string,
+
+  /** The default active key that is expanded on start */
+  defaultActiveKey: PropTypes.string,
+};
+
+const defaultProps = {
+  as: 'div',
+};
+
+const Accordion = React.forwardRef((props, ref) => {
+  let {
+    as: Component,
+    activeKey,
+    bsPrefix,
+    children,
+    className,
+    onClick,
+    ...controlledProps
+  } = useUncontrolled(props, {
+    activeKey: 'onClick',
+  });
+
+  bsPrefix = useBootstrapPrefix(bsPrefix, 'accordion');
+
+  return (
+    <AccordionContext.Provider value={{ onClick, activeEventKey: activeKey }}>
+      <SelectableContext.Provider value={null}>
```

Comment:
i don't think this is needed; the reasons per https://github.com/react-bootstrap/react-bootstrap/blob/89693c7967594d4713149a17aca0993b45373cc0/src/TabPane.js#L167-L171 don't apply here

unless you want to use `SelectableContext` to pass down `onSelect`... might be more DRY than passing down `onClick` in `AccordionContext`

Reply:
I think I'm gonna go with just using `SelectableContext`, should be good enough for what I'm using it for